zone/mob.cpp

```diff
@@ -1692,151 +1694,77 @@ void Mob::SendIllusionPacket(
-	}
-	else {
-		drakkin_tattoo = in_drakkin_tattoo;
-	}
-
-	if (in_drakkin_details == 0xFFFFFFFF) {
-		drakkin_details = GetDrakkinDetails();
-	}
-	else {
-		drakkin_details = in_drakkin_details;
-	}
-
-	if (in_size <= 0.0f) {
-		size = GetSize();
-	}
-	else {
-		size = in_size;
-	}
+	new_haircolor = (in_haircolor == 0xFF) ? GetHairColor() : in_haircolor;
+	new_beardcolor = (in_beardcolor == 0xFF) ? GetBeardColor() : in_beardcolor;
+	new_eyecolor1 = (in_eyecolor1 == 0xFF) ? GetEyeColor1() : in_eyecolor1;
+	new_eyecolor2 = (in_eyecolor2 == 0xFF) ? GetEyeColor2() : in_eyecolor2;
+	new_hairstyle = (in_hairstyle == 0xFF) ? GetHairStyle() : in_hairstyle;
+	new_luclinface = (in_luclinface == 0xFF) ? GetLuclinFace() : in_luclinface;
+	new_beard = (in_beard == 0xFF) ? GetBeard() : in_beard;
+	new_drakkin_heritage = 
+		(in_drakkin_heritage == 0xFF) ? GetDrakkinHeritage() : in_drakkin_heritage;
+	new_drakkin_tattoo = 
+		(in_drakkin_tattoo == 0xFF) ? GetDrakkinTattoo() : in_drakkin_tattoo;
+	new_drakkin_details = 
+		(in_drakkin_details == 0xFF) ? GetDrakkinDetails() : in_drakkin_details;
+	new_aa_title = in_aa_title;
+	size = (in_size <= 0.0f) ? GetSize() : in_size;
 
 	// Reset features to Base from the Player Profile
 	if (IsClient() && in_race == 0) {
-		race             = CastToClient()->GetBaseRace();
-		gender           = CastToClient()->GetBaseGender();
-		texture          = 0xFF;
-		helmtexture      = 0xFF;
-		haircolor        = CastToClient()->GetBaseHairColor();
-		beardcolor       = CastToClient()->GetBaseBeardColor();
-		eyecolor1        = CastToClient()->GetBaseEyeColor();
-		eyecolor2        = CastToClient()->GetBaseEyeColor();
-		hairstyle        = CastToClient()->GetBaseHairStyle();
-		luclinface       = CastToClient()->GetBaseFace();
-		beard            = CastToClient()->GetBaseBeard();
-		aa_title         = 0xFF;
-		drakkin_heritage = CastToClient()->GetBaseHeritage();
-		drakkin_tattoo   = CastToClient()->GetBaseTattoo();
-		drakkin_details  = CastToClient()->GetBaseDetails();
+		race							= CastToClient()->GetBaseRace();
+		gender							= CastToClient()->GetBaseGender();
+		new_texture = texture			= 0xFF;
+		new_helmtexture = helmtexture	= 0xFF;
+		new_haircolor = haircolor		= CastToClient()->GetBaseHairColor();
+		new_beardcolor = beardcolor		= CastToClient()->GetBaseBeardColor();
+		new_eyecolor1 = eyecolor1		= CastToClient()->GetBaseEyeColor();
+		new_eyecolor2 = eyecolor2		= CastToClient()->GetBaseEyeColor();
+		new_hairstyle = hairstyle		= CastToClient()->GetBaseHairStyle();
+		new_luclinface = luclinface		= CastToClient()->GetBaseFace();
+		new_beard = beard				= CastToClient()->GetBaseBeard();
+		new_aa_title = aa_title			= 0xFF;
+		new_drakkin_heritage 			= drakkin_heritage	
+										= CastToClient()->GetBaseHeritage();
+		new_drakkin_tattoo = 			drakkin_tattoo	
+										= CastToClient()->GetBaseTattoo();
+		new_drakkin_details 			= drakkin_details	
+										= CastToClient()->GetBaseDetails();
```

Comment:
Formatting is crazy here

Reply:
Doing 2 assignments in one statement in both cases.  Updating to be on one line (is that what you seek)?